topo_processor/util/transfer_collection.py

```diff
@@ -59,7 +59,11 @@ def transfer_collection(collection: Collection, target: str):
         json_item = stac_item.to_dict(include_self_link=False)
         write_json(json_item, os.path.join(target, item.collection.title, f"{item.id}.json"))
 
+    # after all items have been processed generate summaries
+    collection.generate_summaries(stac_collection)
```

Comment:
I think `generate_summaries` needs to be called by the `validate` command as well, as that doesn't call `transfer_collection`.